### PR TITLE
Update development dependencies

### PR DESCRIPTION
**Pull request by bertdeblock**
Mostly silences a bunch of logs / warnings / vulnerabilities.

Reviewing commit per commit should make this PR clearer.

### CODE REVIEW

File: node-tests/addon-test.js [hunk 1]
```diff
@@ -1035,7 +1035,7 @@ describe('ember-cli-babel', function() {
         expect(
           output.read()
         ).to.deep.equal({
-          "foo.js": `define(\"foo\", [], function () {\n  \"use strict\";\n\n  Ember.String.camelize('stuff-here');\n});`
+          "foo.js": `define("foo", [], function () {\n  "use strict";\n\n  Ember.String.camelize('stuff-here');\n});`
```
Comment (bertdeblock) [inline node-tests/addon-test.js:1038]: These escapes were marked as unnecessary by ESLint.

File: package.json [hunk 2]
```diff
@@ -30,8 +30,7 @@
   },
   "scripts": {
     "build": "ember build",
-    "changelog": "lerna-changelog",
```
Comment (bertdeblock) [inline package.json:33]: Remnant of an older release setup.

File: tests/dummy/app/controllers/application.js [hunk 3]
```diff
@@ -7,10 +10,10 @@ export default class ApplicationController extends Controller {
   constructor() {
     super(...arguments);
 
-    this.animal = new Animal('dog');
+    set(this, 'animal', new Animal('dog'));
```
Comment (bertdeblock) [inline tests/dummy/app/controllers/application.js:13]: Fixes by `eslint-plugin-ember`.

File: tests/dummy/config/targets.js [hunk 4]
```diff
@@ -6,13 +6,6 @@ const browsers = [
   'last 1 Safari versions'
 ];
 
-const isCI = !!process.env.CI;
```
Comment (bertdeblock) [inline tests/dummy/config/targets.js:9]: Fixes the `Uncaught TypeError: (0 , _emberBabel.extends) is not a function` error.

**Comment by kategengler** [comment]
Thank you!

**Pull request merged**